### PR TITLE
fixed error parsing

### LINKED ISSUE #147: No clear error when closing the Talisman popup
start a Tx, then just close the popup. The console error is "Error: Canceled" but it sounds like nothing gets propagated for users.
image

image

### PR DESCRIPTION
**Pull request by uDaiko**
closes #147
Screenshot 2024-10-01 at 20 18 53

### CODE REVIEW

File: src/hooks/useGetSigningCallback.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const useGetSigningCallback =
       },
       error: (error: DispatchError) => {
         console.error(error)
-        toast.error(`Error: ${JSON.stringify(error)}`)
+        toast.error(`${error.toString()}`)
```
Comment (Tbaut) [inline src/hooks/useGetSigningCallback.tsx:37]:
```suggestion
        toast.error(error.toString())
```

**Review by Tbaut — APPROVED** [review]
Thanks, this works well. Please just see my comment, there's no need of the quotes etc any more.

**Comment by uDaiko** [comment]
Ah, such a simple oversight on my part sorry. Thanks for correcting it

**Comment by wirednkod** [comment]
Thank you @uDaiko

**Pull request merged**